registration.html

```diff
@@ -125,6 +125,23 @@ <h3 class="slim-title" for="en-gender">What is your gender?</h3>
 									<input class="slInp hidden" type="text" placeholder="Type your answer here..." id="en-gender">
 								</div>
 							</div>
+							<div class="qGrp">
+								<h3 class="slim-title" for="en-race">What is your race/ethnicity?test</h3>
+								<div class="selectSplit">
+									<select class="slSel" id="en-raceSelect">
+										<option value="select" selected disabled>Select</option>
+										<option value="American Indian/Alaskan Native">American Indian / Alaskan Native</option>
+										<option value="Asian/Pacific Islander">Asian / Pacific Islander</option>
+										<option value="Black/African American">Black / African American</option>
+										<option value="Hispanic">Hispanic</option>
+										<option value="White/Caucasian">White / Caucasian</option>
+										<option value="Multiple Ethnicities">Multiple Ethnicities</option>
+										<option value="Other">Other</option>
+										<option value="decline">Decline to State</option>
```

Comment:
All the rest are capitalized, could you change this value to `"Decline to State"`?